pkgs/development/python-modules/django-debug-toolbar/default.nix

```diff
@@ -0,0 +1,80 @@
+{ lib, fetchFromGitHub
+
+# Build
+, buildPythonPackage
+
+# Runtime
+, django
+, django-jinja
+, sqlparse
+
+# Test
+, coverage
+, flake8
+, html5lib
+, isort
+, selenium
+, tox
+, black
+, pytest
+
+# Documentation
+, sphinx
+, sphinxcontrib-spelling
+}:
+
+buildPythonPackage rec {
+  pname = "django-debug-toolbar";
+  version = "3.2.2";
+
+  src = fetchFromGitHub {
+    owner = "jazzband";
+    repo = pname;
+    rev = version;
+    sha256 = "1dgb3s449nasbnqd5xfikxrfhwwilwlgrw9nv4bfkapvkzpdszjk";
+  };
+
+  nativeBuildInputs = [
+    sphinx
+    sphinxcontrib-spelling
+  ];
+
+  propagatedBuildInputs = [
+    django
+    django-jinja
+    sqlparse
+  ];
+
+  # django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
+  # https://matrix.to/#/!VjfUzaKsXokUdnQcvP:nixos.org/$EsNIzppKZY72CQrM_yfHXPBOoPmZXATUHSVYFiNbnnk?via=nixos.org&via=matrix.org&via=lossy.network
+  # https://github.com/jazzband/django-debug-toolbar/pull/962
+  # https://pyup.io/changelogs/django/#1.11
+  doCheck = false;
+  checkInputs = [
+    coverage
+    flake8
+    html5lib
+    isort
+    selenium
+    tox
+    black
+    pytest
+  ];
+  checkPhase = ''
+    runHook preCheck
+    DJANGO_SETTINGS_MODULE=tests.settings \
+      pytest tests
+    runHook postCheck
+  '';
```

Comment:
All of this can be removed if `doCheck = false`

Reply:
@samuela the checkPhase now is working.